modules/orca/output.tf

```diff
@@ -18,3 +18,24 @@ output "orca_lambda_copy_to_glacier_arn" {
   value = module.orca_lambdas.copy_to_glacier_arn
 }
 
+
+
+
+
+output "orca_sqs_get_message_queue_arn" {
+  value = module.orca_sqs.get_message_queue_arn
```

Comment:
Please add a description to all your new output variables. See comments on resource names and update the outputs accordingly.